jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -634,6 +636,7 @@ private InterpreterResult executeSql(String propertyKey, String sql,
     ResultSet resultSet = null;
     String paragraphId = interpreterContext.getParagraphId();
     String user = interpreterContext.getAuthenticationInfo().getUser();
+    boolean commit = Boolean.valueOf(getProperty(String.format(JDBC_AUTOCOMMIT, propertyKey)));
```

Comment:
I think If old JDBC interpreter setting doesn't have `zeppelin.jdbc.autocommit` option what you created in this PR, it's going to be `false` 
(https://docs.oracle.com/javase/7/docs/api/java/lang/Boolean.html#valueOf(java.lang.String))

Is it the same with current behavior?

Reply:
I've checked it's `false` by default

Reply:
String.format(JDBC_AUTOCOMMIT, propertyKey)

JDBC_AUTOCOMMIT have no markers to replace **propertyKey**